Catappult/appcoins-wallet-android
# [APPC-3908] Fix hidden buttons on on IAP to small devices

**What does this PR do?**

   Fix hidden button on ask someone to pay on small devices.

**Database changed?**

   No

**How should this be manually tested?**

  Try to make a IAP using a small device, when showing all payment methods, the bottom buttons should be completely visible

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-3908](https://aptoide.atlassian.net/browse/APPC-3908)

**Code Review Checklist**



[APPC-3908]: https://aptoide.atlassian.net/browse/APPC-3908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
